packages/flutter_tools/lib/src/build_system/targets/icon_tree_shaker.dart

```diff
@@ -327,15 +327,25 @@ class IconTreeShaker {
       final Object? package = iconDataMap['fontPackage'];
       final Object? fontFamily = iconDataMap['fontFamily'];
       final Object? codePoint = iconDataMap['codePoint'];
-      if ((package ?? '') is! String || // Null is ok here.
-          fontFamily is! String ||
+      if ((package ?? '') is! String ||
+          (fontFamily ?? '') is! String ||
           codePoint is! num) {
         throw IconTreeShakerException._(
           'Invalid ConstFinder result. Expected "fontPackage" to be a String, '
           '"fontFamily" to be a String, and "codePoint" to be an int, '
           'got: $iconDataMap.');
       }
-      final String family = fontFamily;
+      if (fontFamily == null) {
+        _logger.printStatus(
```

Comment:
We might want to just make this a trace if anything. Otherwise every flutter app that transitively depends on this new widget will likely print this message. its likely not actionable.

Reply:
Done thanks!